pkgs/applications/office/gtg/default.nix

```diff
@@ -0,0 +1,73 @@
+{ stdenv
+, fetchFromGitHub
+, meson
+, python3Packages
+, pkgconfig
+, ninja
+, gtk3
+, wrapGAppsHook
+, glib
+, itstool
+, gettext
+, pango
+, gdk-pixbuf
+, gobject-introspection
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "gtg";
+  version = "0.4";
+
+  src = fetchFromGitHub {
+      owner = "getting-things-gnome";
+      repo = "gtg";
+      rev = "6623731f301c1b9c7b727e009f4a6462ad381c68";
+      sha256 = "14gxgg4nl0ki3dn913041jpyfhxsj90fkd55z6mmpyklhr8mwss1";
```

Comment:
Too much indentation.

Reply:
Done, will be in the next pull-request with the other changes.